docs/modules/ROOT/pages/relay.adoc

```diff
@@ -98,6 +98,16 @@ In addition to sending transactions, the Relayer can also sign arbitrary message
 const signResponse = await relayer.sign({ message });
 ----
 
+[[relayer-info]]
+=== Relayer Info
+
+A relayer's address can be retrieved using the `getAddress` method. Checkout the `getRelayer` method of the client to get more info about a relayer.
```

Comment:
I can't find the getAddress method on the relayer object. Maybe you mixed up with the ethers signer object instead?

https://github.com/OpenZeppelin/defender-client/blob/master/src/relayer.ts

Reply:
It is a poor choice of words.